app/scripts/controllers/checkout.js

```diff
@@ -1,5 +1,34 @@
 'use strict';
 
-app.controller('CheckoutCtrl', function($scope, Basket) {
+app.controller('CheckoutCtrl', function($scope, Basket, Order, AuthenticationService) {
   $scope.basket = Basket;
+  $scope.currentUser = AuthenticationService.currentUser();
+  var hour = 60 * 60 * 1000; // 1 Hour in ms
+  $scope.pickupTime = new Date() + (2 * hour);
+
+
+  $scope.createOrder = function() {
+    console.log('Building the order...');
+    var orderItems = [];
+    angular.forEach($scope.basket.get(), function(item) {
+      orderItems.push({'menu_id': item.menu, 'menu_item': item.id, 'quantity': item.quantity});
+    });
+
+    // var order = null;
+    var order = {
+      'order': {
+        // 'user_id': $scope.currentUser.id,
+        // 'menu_id': 2,
+        'order_time': new Date(),
```

Comment:
is there a reason why the `order_time` is sent from the frontend? can't we use the current time from the server? (`created_at` on order record)